### ISSUE TITLE
grad_norm = inf from Epoch 6 While Training from Scratch on KITTI

### ISSUE THREAD

**Issue by okhahmed** [zhenyuw16/Uni3DETR#17]
Hello, thanks for the great work!

I started training the model from scratch on the KITTI dataset using only for (Car, Pedestrian, Cyclist), with the same hyperparameters mentioned in the Uni3DETR paper (lr=9e-5, batch size, etc.) and the provided config file. However, starting from epoch 6, I noticed that the grad_norm becomes inf and continues to increase gradually with each epoch, even though the loss remains stable and the model seems to be learning.

Is this behavior expected, or is there something I might be missing?

Thanks in advance!

**Comment by okhahmed** [comment]
@zhenyuw16

**Comment by okhahmed** [comment]
for KITTI 3 classes training from scratch i used below config

batch size 8 
lr = 9e-5
optimizer = dict(type='AdamW', lr=lr, weight_decay=0.01)
optimizer_config = dict(grad_clip=dict(max_norm=10, norm_type=2))

lr_config = dict(policy='step', warmup=None, step=[64])
runner = dict(type='EpochBasedRunner', max_epochs=70)

i got below mAP 
Pedestrian AP11@0.50, 0.50, 0.50:
bbox AP11:71.5958, 67.3774, 62.8783
bev  AP11:65.2326, 58.3923, 52.7157
3d   AP11:61.1707, 54.7904, 49.1359
aos  AP11:69.73, 65.25, 60.57

Cyclist AP11@0.50, 0.50, 0.50:
bbox AP11:88.9125, 78.7863, 75.9662
bev  AP11:84.8707, 71.0606, 67.7170
3d   AP11:81.8687, 68.7065, 64.4859
aos  AP11:88.81, 78.61, 75.79

Car AP11@0.70, 0.70, 0.70:
bbox AP11:89.4552, 88.9995, 80.5335
bev  AP11:87.3557, 78.7778, 78.4409
3d   AP11:84.7167, 76.2906, 68.5567
aos  AP11:89.44, 88.89, 80.41

Overall AP11@easy, moderate, hard:
bbox AP11:83.3212, 78.3877, 73.1260
bev  AP11:79.1530, 69.4102, 66.2912
3d   AP11:75.9187, 66.5958, 60.7262
aos  AP11:82.66, 77.58, 72.26


can you please share the exact config details? because in the paper mAP for Car is 86%. Thank you.
